cookbook/custom-urls.rst

```diff
@@ -0,0 +1,82 @@
+How to manage custom-urls?
+==========================
+
+Sulu gives the content-manger an easy way to manage custom-urls or landing
+pages. This urls can be defined in the webspace-settings. A custom-url is
+basically an additional url for a specific page.
+
+A custom-url consists of following properties:
+
+
+.. list-table::
+    :header-rows: 1
+
+    * - Parameter
+      - Description
+    * - Title
+      - This title will be displayed in the list and should identify the usecase
+        of this url.
+    * - Activated
+      - This flag indicates the state of the custom-url.
+    * - Base Domain
+      - The content-manager can choose one of the configured base-domains.
+    * - Custom-Url
+      - After choosing the base-domain the ``*`` can be replaced with any value.
+    * - Target page and localization
+      - The selected page will be displayed in the selected localization on this
+        custom-url.
+    * - Redirect
+      - The custom URL will be redirected to URL of the the linked page.
+    * - Canonical URL
+      - When the Canonical URL has been activated only the linked page will be
+        indexed by search engines.
+    * - No Index
+      - The page will not be indexed by search engines.
+    * - No Follow
+      - The links on the page will not be followed by search engines.
+
+.. note::
+
+    If the target page gets deleted the custom-url will stay in the list, will
+    be marked and returns a 404 on the website.
+
+How to configure custom-urls?
+-----------------------------
+
+The custom-urls can be configured in the ``webspace.xml`` in the environment.
```

Comment:
I would explain that these values have impact on the `Base Domain` dropdown mentioned previously.